app/api/routers/text_automatic.py

```
@@ -0,0 +1,76 @@
+            "shop_url": request.shopUrl,
+            "shop_id": request.shopId,
+            "shop_type": request.shopType,
+            "merchant_id": request.merchantId,
+            "platform_integrations": request.platformIntegrations,
+            "reseller_id": request.resellerId,
+        }
+
+        logger.info(f"Calling pipeline_manager.process_message for session {session_id}")
+        # Process message through pipeline manager
+        response_generator = await pipeline_manager.process_message(session_id, request.message, config)
+        logger.info(f"Got response_generator, returning StreamingResponse")
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)